extra_geom/detectors.py

```diff
@@ -1342,3 +1346,131 @@ def __init__(self, modules, filename='No file'):
         warnings.warn(
             "DSSC_Geometry has been renamed to DSSC_1MGeometry.", stacklevel=2
         )
+
+
+class JUNGFRAU_Geometry(DetectorGeometryBase):
```

Comment:
That should be `JUNGFRAUGeometry` to be consistent, right?

Reply:
Yup, I agree. The `_` in the others is standing in for the dash in e.g. 'AGIPD-1M'. We don't need that for JUNGFRAU.

Reply:
Corrected. See comment above.